BoxContentSDK/BoxContentSDK/Operations/BOXAPIMultipartToJSONOperation.m

```diff
@@ -132,6 +132,16 @@ - (void)sessionTask:(NSURLSessionTask *)sessionTask processIntermediateData:(NSD
     }
 }
 
+- (void)sessionTask:(NSURLSessionTask *)sessionTask didFinishWithResponse:(NSURLResponse *)response responseData:(NSData *)responseData error:(NSError *)error
+{
+    @synchronized (self) {
+        if ([self shouldRunInBackground] == YES) {
+            [[NSFileManager defaultManager] removeItemAtPath:self.uploadMultipartCopyFilePath error:nil];
```

Comment:
we won't be able to retry upload using the same operation once it's finished. User can create a new request and the temp file will be re-created.